depthmapX/3DView.cpp

```diff
@@ -365,17 +365,17 @@ void Q3DView::ReloadLineData()
       SuperSpacePixel& superspacepix = *(pDoc->m_meta_graph);
 
       prefvec<Line> lines;
-      for (int i = 0; i < superspacepix.size(); i++) {
-         for (int j = 0; j < superspacepix.at(i).size(); j++) {
-            if (superspacepix.at(i).at(j).isShown()) {
+      for (auto& pixelGroup: superspacepix.m_spacePixels) {

```

Comment:
Should that be `const auto&`? In all or some of the loops?

Reply:
Actually, all the get functions on `pixel` should be const, and also the `setShown` method (the member var whether a pixel is shown needs to be `mutable`, but this is a classic case where this is justified), and then virtually all the loops can use const references.